src/components/PeopleTable.tsx

```diff
@@ -17,628 +27,95 @@ export const PeopleTable = () => {
               </a>
             </span>
           </th>
-
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Sex
-              <a href="#/people?sort=sex">
+              <a href="#/people?sort=name">
```

Comment:
The sort link for the 'Sex' column incorrectly points to `sort=name`. It should be updated to reflect sorting by sex, such as `sort=sex`.